src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1652,6 +1652,7 @@
   "item.minecolonies.amazonegg": "Amazon Spawn Egg",
   "item.minecolonies.amazonchiefegg": "Amazon Leader Spawn Egg",
   "com.minecolonies.coremod.playerinvfull": "You can't pick up this item, as your inventory is full!",
+  "com.minecolonies.coremod.playerinvfull.hotbarinsert": " was dropped in front of you, as your inventory is full!",
```

Comment:
Should this also be for the beekeeper scepter?

Reply:
Done